src/styles/main.scss

```diff
@@ -1,3 +1,52 @@
+@import "reset";
+
 body {
-  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(3, 300px);
+  gap: 0;
+
+  justify-content: center;
+  text-align: center;
+
+  color: white;
+  font-size: 100px;
+  line-height: 300px;
+
+  &__block {
+    height: 300px;
+    width: 300px;
+    background-color: rgb(255, 0, 0);
+  }
+
+  &__block--2 {
+    background-color: rgb(204, 0, 0);
+  }
+
+  &__block--3 {
+    background-color: rgb(153, 0, 0);
+  }
+
+  &__block--4 {
+    background-color: rgb(102, 0, 0);
+  }
+
+  &__block--5 {
+    background-color: rgb(51, 0, 0);
+  }
+
+  &__block--6 {
+    background-color: rgb(0, 0, 0);
+  }
```

Comment:
Try to automate the generation of classes for modifiers (optional)

Reply:
right decision, if we have some more blocks, with my solution we should add more code lines for new blocks. I will use @for statement. Thanks